contracts/vesting/src/tests/instantiate.rs

```diff
@@ -0,0 +1,170 @@
+use soroban_sdk::{
+    testutils::{Address as _, Ledger},
+    vec, Address, Env, String,
+};
+
+use crate::{
+    storage::{DistributionInfo, MinterInfo, VestingBalance, VestingTokenInfo},
+    tests::setup::instantiate_vesting_client,
+};
+
+#[test]
+fn instantiate_contract_succesffully() {
```

Comment:
Again.

```suggestion
fn instantiate_contract_successfully() {
```

Reply:
I'll fix it and every place it occurs. This brings the question — should I include something like [this](https://github.com/marketplace/actions/github-spellcheck-action) in our pipeline.

Reply:
I'd say no need for that... A basic grammar is a must anyway, right.